cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -668,12 +668,12 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
 
         getGroupAccessData: function() {
             var userPartitionId = this.getSelectedEnrollmentTrackId(),
-                groupAccess = {};
-            if (userPartitionId !== 'none' && !isNaN(userPartitionId)) { // If the selected partition ID is not none
```

Comment:
If you want to rebase off of the branch I have since changed this line, we'll need the !isNaN call to verify when the view isn't present that we aren't trying to include a key of NaN in our group_access Object on the post request